sql/09-mutual-likes.sql

```diff
@@ -0,0 +1,8 @@
+-- 09: For every pair of students who both like each other, return the name and grade of both students. Include each pair only once, with the two names in alphabetical order.
+-- Write the SQL in the file sql/09-mutual-likes.sql
+SELECT s1.name, s1.grade, s2.name, s2.grade FROM student_like l1, student_like l2, student s1, student s2
```

Comment:
@eobaah this is awesome that you got it to work, but reading a bit more on `joins` vs `where`, having explicit joins is probably more readable than having implicit joins. Since you are selecting from "multiple tables", postgres's query planner automatically converts them to explicit joins, so the performance of both the queries are the same. The benefit of using explicit joins is readability.